pareto/strategic_water_management/run_strategic_model.py

```diff
@@ -191,6 +192,7 @@
         "water_quality": WaterQuality.false,
         "removal_efficiency_method": RemovalEfficiencyMethod.concentration_based,
         "infrastructure_timing": InfrastructureTiming.true,
+        "subsurface_risk": SubsurfaceRisk.exclude_over_and_under_pressured_wells,
```

Comment:
I cannot comment on line 199, but should we add a doc string which would suggest what the options could be? I believe the following options are valid

- deactivate_slacks
- scale_model
- scaling_factor
- running_time
- gap
- solver
- only_subsurface_block

Not sure if anything else is a part of this options parameter.

Reply:
@JavalVyas2000 I agree it's a good suggestion to add documentation for the options for `solve_model()`. I have added a docstring to the function definition for `solve_model()` in `strategic_produced_water_optimization.py`.

Reply:
Addressed in the solve_model function.